src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -66,7 +68,12 @@ public ActionResult execute() throws EntityDoesNotExistException {
 
             statusToUser.add(new StatusMessage(Const.StatusMessages.COURSE_REMINDER_SENT_TO + studentEmail,
                                                StatusMessageColor.SUCCESS));
-            redirectUrl = Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE;
+
+            if (isRequestedFromPage(previousPage, Const.PageNames.INSTRUCTOR_COURSE_DETAILS_PAGE)) {
```

Comment:
Use `Const.PageNames.INSTRUCTOR_COURSE_DETAILS_PAGE.equals(previousPage)` instead as `previousPage` might be null.

Reply:
To make it clear, can extract a local variable:
```java
boolean isRequestedFromCourseDetailsPage = Const.PageNames.INSTRUCTOR_COURSE_DETAILS_PAGE.equals(previousPage);
redirectUrl = isRequestedFromCourseDetailsPage ? Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE
                                               : Const.ActionURIs.INSTRUCTOR_STUDENT_LIST_PAGE;
```